### DISCUSSION TITLE
Inertia is dead?

### DISCUSSION THREAD

**Discussion by mstfkhazaal** [inertiajs/inertia#1843] [Help]
Inertia is dead?

**Comment by jefhar** [comment]
@mstfkhazaal Do you ask that because there hasn't been a maintenance commit pushed in 6 hours, or because it is ["complete software"](https://twitter.com/taylorotwell/status/1684999340446900224), or do you have some other source?

**Comment by mstfkhazaal** [comment]
Thank you for the clarification.
In my questions, I mean, are there any important updates?
